### PR TITLE
aws_kms - minor doc fixes

### PR DESCRIPTION
**Pull request by jatorcasso**
##### SUMMARY
Minor doc fixes

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
- aws_kms
- aws_kms_info

### CODE REVIEW

File: plugins/modules/aws_kms.py [hunk 1]
```diff
@@ -280,7 +280,9 @@
   returned: always
   sample: arn:aws:kms:ap-southeast-2:123456789012:key/abcd1234-abcd-1234-5678-ef1234567890
 key_state:
-  description: The state of the key
+  description:
+    - The state of the key
+    - Will be one of ['Creating', 'Enabled', 'Disabled', 'PendingDeletion', 'PendingImport', 'PendingReplicaDeletion', 'Unavailable', 'Updating']
```
Comment (tremble) [inline plugins/modules/aws_kms.py:285]:
```suggestion
    - Will be one of C('Creating'), C('Enabled'), C('Disabled'), C('PendingDeletion'), C('PendingImport'), C('PendingReplicaDeletion'), C('Unavailable') or C('Updating')
```

**Review by tremble — CHANGES_REQUESTED** [review]

File: plugins/modules/aws_kms.py (hunk 1 above)
Comment (jatorcasso) [reply plugins/modules/aws_kms.py]: fixed!

**Comment by jatorcasso** [comment]
recheck

File: plugins/modules/aws_kms.py [hunk 2]
```diff
@@ -302,18 +305,23 @@
   returned: always
   sample: 1234567890123
 creation_date:
-  description: Date of creation of the key
+  description: Date and time of creation of the key
   type: str
   returned: always
   sample: "2017-04-18T15:12:08.551000+10:00"
+deletion_date:
```
Comment (alinabuzachis) [inline plugins/modules/aws_kms.py:312]: Don't we need to add version_added?

File: plugins/modules/aws_kms_info.py [hunk 3]
```diff
@@ -116,22 +118,27 @@
       returned: always
       sample: 1234567890123
     creation_date:
-      description: Date of creation of the key
+      description: Date and time of creation of the key
       type: str
       returned: always
       sample: "2017-04-18T15:12:08.551000+10:00"
+    deletion_date:
```
Comment (alinabuzachis) [inline plugins/modules/aws_kms_info.py:125]: Same here?

File: plugins/modules/aws_kms.py (hunk 2 above)
Comment (jatorcasso) [reply plugins/modules/aws_kms.py:312]: This parameter was always returned, it just wasn't documented as being returned. Does it still need a version_added?

Comment (jatorcasso) [reply plugins/modules/aws_kms.py:312]: updated!

**Comment by alinabuzachis** [comment]
recheck

**Comment by jatorcasso** [comment]
Recheck

**Review by markuman — APPROVED** [review]

File: plugins/modules/aws_kms.py [hunk 4]
```diff
@@ -280,7 +280,10 @@
   returned: always
   sample: arn:aws:kms:ap-southeast-2:123456789012:key/abcd1234-abcd-1234-5678-ef1234567890
 key_state:
-  description: The state of the key
+  description:
+    - The state of the key
```
Comment (alinabuzachis) [inline plugins/modules/aws_kms.py:284]:
```suggestion
    - The state of the key.
```

File: plugins/modules/aws_kms.py [hunk 5]
```diff
@@ -302,18 +305,24 @@
   returned: always
   sample: 1234567890123
 creation_date:
-  description: Date of creation of the key
+  description: Date and time of creation of the key
```
Comment (alinabuzachis) [inline plugins/modules/aws_kms.py:308]:
```suggestion
  description: Date and time of creation of the key.
```

File: plugins/modules/aws_kms.py [hunk 6]
```diff
@@ -302,18 +305,24 @@
...
   type: str
   returned: always
   sample: "2017-04-18T15:12:08.551000+10:00"
+deletion_date:
+  description: Date and time after which KMS deletes this KMS key
```
Comment (alinabuzachis) [inline plugins/modules/aws_kms.py:313]:
```suggestion
  description: Date and time after which KMS deletes this KMS key.
```

File: plugins/modules/aws_kms.py [hunk 7]
```diff
@@ -302,18 +305,24 @@
...
+  version_added: 3.3.0
 description:
   description: Description of the key
   type: str
   returned: always
   sample: "My Key for Protecting important stuff"
 enabled:
-  description: Whether the key is enabled. True if C(KeyState) is true.
-  type: str
+  description: Whether the key is enabled. True if I(key_state) is C(Enabled).
```
Comment (alinabuzachis) [inline plugins/modules/aws_kms.py:324]:
```suggestion
  description: Whether the key is enabled. C(True) if I(key_state) is Enabled.
```

File: plugins/modules/aws_kms.py [hunk 8]
```diff
@@ -400,15 +409,16 @@
       Resource: "*"
   version_added: 3.3.0
 tags:
-  description: dictionary of tags applied to the key
+  description: dictionary of tags applied to the key. Empty when access is denied even if there are tags.
```
Comment (alinabuzachis) [inline plugins/modules/aws_kms.py:412]:
```suggestion
  description: Dictionary of tags applied to the key. Empty when access is denied even if there are tags.
```

File: plugins/modules/aws_kms_info.py [hunk 9]
```diff
@@ -93,7 +93,10 @@
       returned: always
       sample: arn:aws:kms:ap-southeast-2:123456789012:key/abcd1234-abcd-1234-5678-ef1234567890
     key_state:
-      description: The state of the key
+      description:
+        - The state of the key
```
Comment (alinabuzachis) [inline plugins/modules/aws_kms_info.py:97]:
```suggestion
        - The state of the key.
```

File: plugins/modules/aws_kms_info.py [hunk 10]
```diff
@@ -93,7 +93,10 @@
...
+        - Will be one of C('Creating'), C('Enabled'), C('Disabled'), C('PendingDeletion'), C('PendingImport'),
+          C('PendingReplicaDeletion'), C('Unavailable'), or C('Updating')
```
Comment (alinabuzachis) [inline plugins/modules/aws_kms_info.py:99]: You can also use `choices`.
```suggestion
          C('PendingReplicaDeletion'), C('Unavailable'), or C('Updating').
```

File: plugins/modules/aws_kms.py [hunk 11]
```diff
@@ -280,7 +280,10 @@
   returned: always
   sample: arn:aws:kms:ap-southeast-2:123456789012:key/abcd1234-abcd-1234-5678-ef1234567890
 key_state:
-  description: The state of the key
+  description:
+    - The state of the key
+    - Will be one of C('Creating'), C('Enabled'), C('Disabled'), C('PendingDeletion'), C('PendingImport'),
+      C('PendingReplicaDeletion'), C('Unavailable'), or C('Updating')
```
Comment (alinabuzachis) [inline plugins/modules/aws_kms.py:286]: You can also use `choices`
```suggestion
      C('PendingReplicaDeletion'), C('Unavailable'), or C('Updating').
```

File: plugins/modules/aws_kms_info.py [hunk 12]
```diff
@@ -116,22 +118,28 @@
       returned: always
       sample: 1234567890123
     creation_date:
-      description: Date of creation of the key
+      description: Date and time of creation of the key
```
Comment (alinabuzachis) [inline plugins/modules/aws_kms_info.py:121]:
```suggestion
      description: Date and time of creation of the key.
```

File: plugins/modules/aws_kms_info.py [hunk 13]
```diff
@@ -116,22 +118,28 @@
...
       type: str
       returned: always
       sample: "2017-04-18T15:12:08.551000+10:00"
+    deletion_date:
+      description: Date and time after which KMS deletes this KMS key
```
Comment (alinabuzachis) [inline plugins/modules/aws_kms_info.py:126]:
```suggestion
      description: Date and time after which KMS deletes this KMS key.
```

File: plugins/modules/aws_kms_info.py [hunk 14]
```diff
@@ -116,22 +118,28 @@
...
+      version_added: 3.3.0
     description:
       description: Description of the key
       type: str
       returned: always
       sample: "My Key for Protecting important stuff"
     enabled:
-      description: Whether the key is enabled. True if C(KeyState) is true.
-      type: str
+      description: Whether the key is enabled. True if I(key_state) is C(Enabled).
```
Comment (alinabuzachis) [inline plugins/modules/aws_kms_info.py:137]:
```suggestion
      description: Whether the key is enabled. C(True) if I(key_state) is Enabled.
```

**Review by alinabuzachis — CHANGES_REQUESTED** [review]

File: plugins/modules/aws_kms_info.py (hunk 10 above)
Comment (jatorcasso) [reply plugins/modules/aws_kms_info.py:99]: @alinabuzachis can you use choices for a return val?

Comment (alinabuzachis) [reply plugins/modules/aws_kms_info.py]: Oh, I realised now it is a returned val. I don't think so.

**Review by alinabuzachis — APPROVED** [review]

**Review by tremble — APPROVED** [review]

**Pull request merged**